apps/dashboard/config/configuration_singleton.rb

```diff
@@ -37,7 +37,7 @@ def initialize
   # @return [Hash] key/value pairs of defaults
   def boolean_configs
     {
-      :csp_enabled                  => false,
+      :csp_enabled                  => true,
```

Comment:
This is just for testing, will revert shortly

Reply:
Yea i figured it was in progress. In any case - I may not get to for a couple days. I have your other PR and others to look over.